modules/platforms/cpp/ignite/client/sql/column_metadata.h

```diff
@@ -18,7 +18,7 @@
 #pragma once
 
 #include "ignite/client/sql/column_origin.h"
-#include "ignite/client/sql/sql_column_type.h"
+#include "ignite/schema/ignite_type.h"
```

Comment:
This is not good. `schema` is not a public library for client, so removal of `sql_column_type` here should be reversed.